DFC.ServiceTaxonomy.GraphSync/Views/GraphSyncPart.Edit.cshtml

```diff
@@ -7,3 +7,13 @@
     <input asp-for="Text" class="form-control content-preview-text" readonly>
     <span class="hint">This URI ID is automatically generated</span>
 </fieldset>
+
+@if (Model.Readonly)
+{
+    <script depends-on="jQuery" at="Foot">
+        $(function () {
+            $('input').attr('readonly', 'readonly');
```

Comment:
This needs to be more targeted. It makes the filter input in the menu read only too.